themes/default/content/blog/meals-for-ukraine/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Teaming up to help deliver more meals to Ukraine"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-05-16T09:00:54+02:00
+meta_desc: Buoyant, Civo, Cockroach Labs, Fairwinds, Pulumi, & Jetstack team up to donate to World Central Kitchen to provide meals in response to humanitarian crises.
+meta_image: kubecon-ukraine.png
+authors:
+    - wendy-smith
+tags:
+    - community
+    - pulumi-events
+---
+Why not do good while learning about cloud native tech? [Buoyant](https://buoyant.io/), [Civo](https://www.civo.com/), [Cockroach Labs](https://www.cockroachlabs.com/), [Fairwinds](https://www.fairwinds.com/), [Pulumi](https://www.pulumi.com/), and [Jetstack](https://www.jetstack.io/) have teamed up to donate $2 for each virtual booth visitor to [World Central Kitchen](https://wck.org/) (WCK), a non-profit committed to providing meals in response to humanitarian crises.
+
+<!--more-->
+Working with restaurants across Ukraine and in six countries welcoming refugees, WCK is providing hundreds of thousands of daily meals for people in need. WCK serves meals at shelters, hospitals, transit hubs, and more. It's a wonderful mission and, if you agree, we encourage you to [donate to WCK directly](https://donate.wck.org/give/398293#!/donation/checkout) 💙 🇺🇦 💛
+
+## Tweets
+
+{{< tweet id="1510622155775619078" user="pulumipus" >}}
+
+## How it works
+
+Participating is easy. If you registered for KubeCon, just click on the booth links below to visit our virtual booths. Each visit will count as a $2 donation, so if you visit us all, that's $12 for the cause right there!
+
+Of course, there are many more reasons to visit our booths. You'll find lots of great cloud native resources including demos, tutorials, ebooks, and more. Here are some of the highlights you can expect at each booth.
+
+### Pulumi — Universal Infrastructure as Code
+
+At [Pulumi](https://www.pulumi.com/)’s booth, you'll learn about their cloud engineering platform that brings infrastructure, developer, and security teams together through a unified software engineering process that tames cloud complexity and accelerates innovation. Don't miss Pulumi's Ask an Expert live sessions to hear directly from the engineering team. Whether you want to know more about how to build with Universal Infrastructure as Code in general-purpose programming and markup languages or just learn more about what's on the roadmap, this is your chance to ask anything. [Visit Pulumi today](https://bit.ly/3M5b6Qj)!
```

Comment:
Suggest a slight tweak, up to you:

```suggestion
At the [Pulumi](https://www.pulumi.com/) booth, you'll learn about our cloud engineering platform, which brings infrastructure, developer, and security teams together through a unified software engineering process that tames cloud complexity and accelerates innovation. Don't miss our Ask an Expert live sessions to hear directly from the engineering team. Whether you want to know more about how to build with Universal Infrastructure as Code in general-purpose programming and markup languages or just learn more about what's on the roadmap, this is your chance to ask us anything. [Visit the Pulumi booth today](https://bit.ly/3M5b6Qj)!
```

Also, these links seems to require sign-in:

![image](https://user-images.githubusercontent.com/274700/168170277-baccbadf-539d-4da1-b110-c2f4113532fc.png)

Should we mention somewhere briefly (probably under How it Works) that you'll need to be signed in and/or be registered to visit the booth links?

Reply:
I think it says that at the top when there’s something about being registered for Kubecon?

also these links won’t work even with sign in until Monday i don’t think.